test/data/navigator/useragentdata-brand.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<!-- navigator.userAgentData brand test -->
+<html>
+  <head>
+    <title>fail</title>
+    <meta charset="utf-8">
+</head>
+<body>
+  <script>
+    navigator.userAgentData.getHighEntropyValues(
```

Comment:
can we execute this script from the test cpp so it's easier to see what it's running without having to look at the test page itself?

Reply:
fixed in https://github.com/brave/brave-core/commit/d186816a8ec15adde1d723ae3bdf1664db604041